src/ConstructiveSolidGeometry/VolumePrimitives/Box.jl

```diff
@@ -70,6 +70,15 @@ function Box(CO, hX::TX, hY::TY, hZ::TZ, origin::PT, rotation::ROT) where {TX, T
     Box{T}(CO,T(hX), T(hY), T(hZ), origin, rotation)
 end
 
+#Unit convesion
+Box(CO, hX::Quantity, hY, hZ, origin, rotation) = Box(CO, to_internal_units(hX), hY, hZ, origin, rotation)
```

Comment:
Is there a convenient way to write those permutations?

Reply:
Use `RealQuantity`?

Reply:
Is `RealQuantity` part of Unitful? I can not find it

Reply:
`RealQuantity` is part of SSD:
https://github.com/JuliaPhysics/SolidStateDetectors.jl/blob/9dabbf63f698fb1c73b03f8b1fd7218fdf1a1e23/src/Units.jl#L90